.env

```diff
@@ -1,40 +1,38 @@
-REACT_APP_AGGREGATOR_API=https://meta-aggregator.dev.kyberengineering.io
-REACT_APP_AGGREGATOR_STATS_API=https://aggregator-stats.dev.kyberengineering.io
+REACT_APP_AGGREGATOR_API=https://meta-aggregator.stg.kyberengineering.io
+REACT_APP_AGGREGATOR_STATS_API=https://aggregator-stats.kyberswap.com
 REACT_APP_SENTRY_DNS=https://d94ee2d3c22043bdaec966758680b5a8@sentry.ops.kyberengineering.io/4
 
 GENERATE_SOURCEMAP=false
-REACT_APP_SERVICE_WORKER=true
+REACT_APP_SERVICE_WORKER=false
 
-REACT_APP_TRUESIGHT_API=https://truesight.dev.kyberengineering.io
-REACT_APP_MIXPANEL_PROJECT_TOKEN=fca28a30cb98d872c2079f214955cd5e
-REACT_APP_REWARD_SERVICE_API=https://reward.dev.kyberengineering.io/api/v1
+REACT_APP_TRUESIGHT_API=https://truesight.stg.kyberengineering.io
+REACT_APP_MIXPANEL_PROJECT_TOKEN=ff1eea26c19dcf4a7c35ebbc8631e714
+REACT_APP_REWARD_SERVICE_API=https://reward.stg.kyberengineering.io/api/v1
 
-REACT_APP_PRICE_CHART_API=https://price-chart.dev.kyberengineering.io/api
-REACT_APP_PRICE_API=https://price.dev.kyberengineering.io
+REACT_APP_PRICE_CHART_API=https://price-chart.kyberswap.com/api
+REACT_APP_PRICE_API=https://price.stg.kyberengineering.io
 
-REACT_APP_NOTIFICATION_API=https://notification.dev.kyberengineering.io/api
+REACT_APP_NOTIFICATION_API=https://notification.stg.kyberengineering.io/api
 
-REACT_APP_CAMPAIGN_BASE_URL=https://campaigns.dev.kyberengineering.io
-REACT_APP_TYPE_AND_SWAP_URL=https://type-swap.dev.kyberengineering.io/api
+REACT_APP_CAMPAIGN_BASE_URL=https://campaigns.stg.kyberengineering.io
+REACT_APP_TYPE_AND_SWAP_URL=https://type-swap.stg.kyberengineering.io/api
 
 REACT_APP_TRANSAK_URL=https://staging-global.transak.com
 REACT_APP_TRANSAK_API_KEY=327b8b63-626b-4376-baf2-70a304c48488
 
-REACT_APP_KS_SETTING_API=https://ks-setting.dev.kyberengineering.io/api
+REACT_APP_KS_SETTING_API=https://ks-setting.stg.kyberengineering.io/api
 
 REACT_APP_GTM_ID=
 
 REACT_APP_GOOGLE_RECAPTCHA_KEY=6Lc7yg0iAAAAALhS4C4Ez5hqMBub8hKdFvvr9sfc
 REACT_APP_POOL_FARM_BASE_URL=https://pool-farm.kyberswap.com
 
-REACT_APP_LIMIT_ORDER_API_WRITE=https://limit-order.dev.kyberengineering.io/write/api
-REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.dev.kyberengineering.io/read-ks/api
+REACT_APP_LIMIT_ORDER_API_WRITE=https://limit-order.stg.kyberengineering.io/write/api
+REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.stg.kyberengineering.io/read-ks/api
 
 # Kyber DAO Apis
-REACT_APP_KYBER_DAO_STATS_API=https://kyberswap-dao-stats.kyberengineering.io
-
-REACT_APP_PRICE_ALERT_API=https://price-alert.dev.kyberengineering.io/api
-
+REACT_APP_KYBER_DAO_STATS_API=https://kyberswap-dao-stats.stg.kyberengineering.io
+REACT_APP_PRICE_ALERT_API=https://price-alert.stg.kyberengineering.io/api
```

Comment:
todo: let's revert after testing finished

Reply:
Will do bro

Reply:
should revert now?